data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,102 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+mission "Hai Pilgrimage"
+	name "Hai spiritual leader to <planet>"
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
+	minor
+	clearance "You inform the authorities of the passenger you are transporting, and they quickly allow you to land."
+	source
+		government "Hai"
+		not attributes "station"
+	destination
+		attributes "unfettered"
```

Comment:
Seeing as this mission gives you 100 unfettered rep, I think the extra challenge of having to travel through unfettered space is justified.

Reply:
This looks like it can select planets in Wanderer space after the Unfettered Hai capture them. Is that something you want?

Reply:
Oh that'd be the only way to land on them lmao

Reply:
> This looks like it can select planets in Wanderer space after the Unfettered Hai capture them. Is that something you want?

I think it's fine. Would fit into the whole 'big changes are coming' message Hareek has.